proto/proto-backplane-grpc/src/main/proto/io/deephaven/proto/console.proto

```diff
@@ -201,9 +201,263 @@ message FetchFigureRequest {
     io.deephaven.proto.backplane.grpc.Ticket console_id = 1;
     reserved 2;
 
+    string figure_name = 3;
 }
 message FetchFigureResponse {
+    FigureDescriptor figure_descriptor = 1;
+}
+
+//TODO before merging this, redo the field ids
+message FigureDescriptor {
+    optional string title = 1;
+    string title_font = 2;
+    string title_color = 3;
+
+    int64 update_interval = 7;
+
+    int32 cols = 8;
+    int32 rows = 9;
+
+    repeated ChartDescriptor charts = 10;
+
+    repeated io.deephaven.proto.backplane.grpc.ExportedTableCreationResponse tables = 11;
+
+    reserved 13;// reserved for tablemaps, once those are well supported
+//    repeated io.deephaven.proto.backplane.grpc.Ticket table_maps = 13;
+
+    repeated string errors = 15;
+
+    message ChartDescriptor {
+        enum ChartType {
+            XY = 0;
+            PIE = 1;
+            OHLC = 2;
+            CATEGORY = 3;
+            XYZ = 4;
+            CATEGORY_3D = 5;
+        }
+        int32 colspan = 1;
+        int32 rowspan = 2;
+
+        repeated SeriesDescriptor series = 3;
+        repeated MultiSeriesDescriptor multi_series = 4;
+        repeated AxisDescriptor axes = 5;
+
+        ChartType chart_type = 6;
+
+        optional string title = 7;
+        string title_font = 8;
+        string title_color = 9;
+
+        bool show_legend = 10;
+        string legend_font = 11;
+        string legend_color = 12;
+
+        bool is3d = 13;
+    }
+
+    enum SeriesPlotStyle {
+        BAR = 0;
+        STACKED_BAR = 1;
+        LINE = 2;
+        AREA = 3;
+        STACKED_AREA = 4;
+        PIE = 5;
+        HISTOGRAM = 6;
+        OHLC = 7;
+        SCATTER = 8;
+        STEP = 9;
+        ERROR_BAR = 10;
+    }
+
+    message SeriesDescriptor {
+        SeriesPlotStyle plot_style = 1;
+
+        string name = 2;
+
+        optional bool lines_visible = 3;
+        optional bool shapes_visible = 4;
+
+        bool gradient_visible = 5;
+        string line_color = 6;
+        //    string line_style = 7;
```

Comment:
Because the feature isn't used in enterprise either. I'll add a `reserved` for it in case we ever decide to add it, and a note to a follow up issue in core to consider the other missed features.